### DISCUSSION TITLE
Crazyflie CF Stuck on Solid Blue M2 LED.

### DISCUSSION THREAD

**Discussion by Wallat1** [bitcraze/discussions#1206] [Q&A]
Hi.

On Monday I was able to update my crazyflies to the firmware version 2020.6 cf or other without problem, with the cfclient version 2022.3.post20+63f0516, however on Tuesday when I reloaded the firmware on one of my crazyflies it only kept the LED M2 stuck, to check if it was a problem with my crazyflie I tried another one and the same thing happened now I have 4 crazyflies in the same state, for this reason I would like to know if  are you  having problems with the cfclient or what is happening,? I would appreciate your help.

Regards.

**Comment by knmcguire** [comment]
Hi!

This is a real old version of the crazyflie and the cfclient so we can't really help unless you want to use the updated one.

You can use recovery mode to fix this issue: https://www.bitcraze.io/documentation/repository/crazyflie-clients-python/master/userguides/recovery-mode/

**Reply by Wallat1** [reply]
Hi

Thank you for your quick response, but this happens to me even when loading the 2023.11 cf version and when I load the 2024.2 cf the crazyflie only restarts and remains with the M2 led stuck.

Regards.

**Reply by knmcguire** [reply]
Ah oke. Could you update your version of your client with

`pip3 install cfclient --upgrade` and try again? Your cfclient version is 2 years old and we have introduced some fixes in the new cfclient to be able to update to the newest version of the crazyflie firmware.

**Reply by Wallat1** [reply]
Hi.

I have executed pip3 install cfclient --upgrade and now I can install version 2024.2 and the M2 led does not stay stuck, but I cannot use versions prior to said versions, I think you have some error in the cfclient installer, since now I have the same problem on 3 computers with Ubuntu 20.04 and two with W10, and thanks to this I cannot continue with my work and version 2024.2 is not useful to me. I'd appreciate your help.

Regards.

**Reply by knmcguire** [reply]
Alright, at least you were able to recover the firmware versio. 

About your client issue, I see that one of my colleague is already helping out in another thread so that is perhaps better to further the discussion there: https://github.com/orgs/bitcraze/discussions/1206#discussioncomment-8674242

**Reply by Wallat1** [reply]
Hi.

Thanks for your attention, I have already been able to solve the problem, versions 3.8 and 3.9 of python do not work well with the new version of cfclient, so I had to use the versions 3.11 and 3.12.

Regards.
